plugin/federation/testdata/federation2/federation2.graphql

```diff
@@ -1,13 +1,13 @@
 extend schema
-  @link(url: "https://specs.apollo.dev/federation/v2.3",
-        import: ["@key", "@shareable", "@provides", "@external", "@tag", "@extends", "@override", "@inaccessible", "@interfaceObject"])
+  @link(url: "https://specs.apollo.dev/federation/v2.7",
+        import: ["@key", "@shareable", "@provides", "@external", "@tag", "@extends", "@override", "@inaccessible", "@interfaceObject", "Policy"])
```

Comment:
```suggestion
        import: ["@key", "@shareable", "@provides", "@external", "@tag", "@extends", "@override", "@inaccessible", "@interfaceObject", "@policy"])
```

Isn't this suppose to be `@policy` ?

Reply:
[b33376a](https://github.com/99designs/gqlgen/pull/2876/commits/b33376a71e320af6c2fb455bd631d3ff9d152585)